### ISSUE TITLE
WCAG compliant color palette

### ISSUE THREAD

**Issue by Steinop** [fremtind/jokul#301]
Update colour palette - Green, blue, red, etc to become WCAG compliant and add a wider range of grays that will work for dark / light themes. Decide if we should have a specialised colour palette for UX colours.

**Comment by Steinop** [comment]
Daniel has done some sketches round this matter. Check with him.

**Comment by Steinop** [comment]
More regarding colors: https://uxdesign.cc/my-struggle-with-colors-part-ii-ed71bff6302a

**Comment by Mikkelinski** [comment]
@Steinop link to Figma file with considered colours?

**Comment by Steinop** [comment]
@Mikkelinski Check with Hilde. Can not find her work area.

**Comment by Steinop** [comment]
Figma reference: https://www.figma.com/file/TkbB9ANfejDSjB2u4u1OEuqM/J%C3%B8kul-komponenter-byggesone?node-id=3335%3A3156

**Comment by Mikkelinski** [comment]
This will solve #372 once it is completed
